raphtory-graphql/src/model/mod.rs

```diff
@@ -192,17 +212,21 @@ impl Mut {
         Ok(true)
     }
 
-    // If namespace is not provided, it will be set to the current working directory.
-    // This applies to both the graph namespace and new graph namespace.
+    /// Move graph from a path path on the server to a new_path on the server.
+    ///
+    /// If namespace is not provided, it will be set to the current working directory.
+    /// This applies to both the graph namespace and new graph namespace.
```

Comment:
I think these namespace comments referr to when we had namespaces and graph names seperate, this no longer makes sense

Reply:
Also how does this fail?